docs/MO_DG/prepare_model/convert_model/Additional_Optimizations.md

```diff
@@ -0,0 +1,48 @@
+# Optimizing Preprocessing Computation {#openvino_docs_MO_DG_Additional_Optimization_Use_Cases}
+
+Input data for inference can be different from training dataset and requires additional preprocessing before inference.
```

Comment:
Why you changing this file so much if it being reworked in a different PR #10820 ?

Reply:
discussed offline